deep-learning/_posts/2018-09-27-the_elephant_in_the_room.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: review
+title: "The Elephant in the Room"
+tags: deep-learning
+author: "Antoine Théberge"
+cite:
+    authors: "Amir Rosenfeld, Richard Zemel and John K. Tsotsos"
+    title:   "The Elephant in the Room"
+    venue:   "Arxiv"
+pdf: "https://arxiv.org/abs/1808.03305"
+---
+
+## The idea
+Adversarial examples have been suggested as small targeted perturbations imperceptible to the human eye. This paper does the opposite and transplant objects, from the same "dataset", into another image. This has multiple undesirable effects.
+
+## The method 
+All the experiments used images from COCO 2017's validation set. Models are from Tensorflow's Object Detection API. Various metrics were used to calculate how much object detectors failed.
+
+The first test was to extract an object (a elephant, wink) from another image and transplant it at various locations in another image (a living room, wink) on which the detector was trained (Faster-RCNN with NASNet backbone). Results:
+
+![](/deep-learning/images/the_elephant_in_the_room/litteral-elephant.png)
+![](/deep-learning/images/the_elephant_in_the_room/polar-bear.png)
+![](/deep-learning/images/the_elephant_in_the_room/sandwich.png)
+![](/deep-learning/images/the_elephant_in_the_room/bus.png)
+![](/deep-learning/images/the_elephant_in_the_room/horse.png)
+![](/deep-learning/images/the_elephant_in_the_room/surfboard.png)
+
```

Comment:
Add a --- to separate those section, not easy to see the text between the images